R/bs-theme-preset-bootswatch.R

```diff
@@ -27,12 +27,10 @@ theme_bootswatch <- function(theme) {
 theme_version <- function(theme) {
   if (!is_bs_theme(theme)) return(NULL)
 
-  version_class <- grep("^bs_version_", class(theme), value = TRUE)
-  if (length(version_class) > 0) {
-    sub("^bs_version_", "", version_class)
-  } else {
-    theme_preset_info(theme)$version
-  }
+  if (inherits(theme, "bs_version_3")) return("3")
+  if (inherits(theme, "bs_version_4")) return("4")
+  if (inherits(theme, "bs_version_5")) return("5")
+  theme_preset_info(theme)$version
```

Comment:
Either write a unit test that fails when we add 6 or go back to what we had before

Reply:
Added a test in 7805b8923aed7254b67cc97bf88add43df89d130